larana/OpticalDetector/OpHitFinder/RiseTimeTools/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cet_enable_asserts()
+
+cet_make_library(LIBRARY_NAME RiseTimeCalculatorTool INTERFACE
+  SOURCE RiseTimeCalculatorBase.h
+)
+
+cet_write_plugin_builder(lar::RiseTimeCalculatorTool art::tool Modules
+  INSTALL_BUILDER
+)
```

Comment:
Although `RiseTimeCalculatorTool` does not currently confer any `INTERFACE` dependencies, it should itself still be a necessary part of any `lar::RiseTimeCalculatorTool`, viz:
```{.cmake}
  LIBRARIES CONDITIONAL larana::RiseTimeCalculatorTool
```